js/main.js

```diff
@@ -31,21 +19,19 @@ function handleURLChange(event) {
 }
 
 function handleFormSubmit(event) {
-  nextEntryId++;
+  data.nextEntryId++;
```

Comment:
You should increment after using this value, which will let you keep the original `nextEntryId` value at 1